addons/project/controllers/portal.py

```diff
@@ -317,6 +317,7 @@ def _task_get_searchbar_groupby(self, milestones_allowed, project=False):
             'state': {'label': _('Status'), 'sequence': 40},
             'priority': {'label': _('Priority'), 'sequence': 60},
             'partner_id': {'label': _('Customer'), 'sequence': 70},
+            'parent_id': {'label': _('Parent task'), 'sequence': 80},
```

Comment:
```suggestion
            'parent_id': {'label': self.env._ ('Parent task'), 'sequence': 80},
```
Ref:https://github.com/odoo/odoo/pull/174844